pkgs/development/libraries/libzen/default.nix

```diff
@@ -1,12 +1,12 @@
 { stdenv, fetchurl, automake, autoconf, libtool, pkgconfig }:
 
-let version = "0.4.32"; in
+let version = "0.4.33"; in
 
 stdenv.mkDerivation {
   name = "libzen-${version}";
```

Comment:
You can omit the `let` above and define `version` right here if you add the `rec` keyword after `mkDerivation`.